src/Microsoft.OData.Core/ODataSimplifiedOptions.cs

```diff
@@ -33,12 +42,12 @@ public ODataSimplifiedOptions(ODataVersion? version)
             if (version == null || version < ODataVersion.V401)
             {
                 this.EnableReadingODataAnnotationWithoutPrefix = false;
-                this.EnableWritingODataAnnotationWithoutPrefix = false;
+                this.enableWritingODataAnnotationWithoutPrefix = this.omitODataPrefix40;
             }
             else
             {
                 this.EnableReadingODataAnnotationWithoutPrefix = true;
-                this.EnableWritingODataAnnotationWithoutPrefix = true;
+                this.enableWritingODataAnnotationWithoutPrefix = omitODataPrefix;

```

Comment:
nit: Should use this.omitODataPrefix here as well?

Reply:
yes; thanks.

---
In reply to: [283528439](https://github.com/OData/odata.net/pull/1454#discussion_r283528439) [](ancestors = 283528439)